src/modules/audio_device/audio_device_impl.cc

```diff
@@ -20,6 +20,11 @@
 #include "rtc_base/ref_counted_object.h"
 #include "system_wrappers/include/metrics.h"
 
+/* In the scope of this file there is no difference between Linux & FreeBSD */
```

Comment:
#ifdef x3

Reply:
It shouldn't be compiled if you set TG_OWT_BUILD_AUDIO_BACKENDS=OFF as recommended per https://github.com/telegramdesktop/tdesktop/wiki/The-Packaged-Building-Mode

Reply:
it is, but WEBRTC_DUMMY_AUDIO_BUILD will be set, so the change shouldn't have any visible result